src/integ_test_resources/flutter/amplify/github-cloudwatch-cdk/lib/createCanaryTestFailureAlarm.ts

```diff
@@ -0,0 +1,93 @@
+import { Duration } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { Alarm, ComparisonOperator, GraphWidget, Metric, TextWidget, TreatMissingData } from 'aws-cdk-lib/aws-cloudwatch';
+
+// Create the metric and alarms for tracking Github Canary failures.
+// These metrics are sent in Github action: amplify-flutter/amplify_canaries
+export function createCanaryTestFailureAlarm(scope: Construct) {
+
+  const buildFailureMetric = new Metric({
+    metricName: "BuildCanaryTestFailure",
+    namespace: "GithubCanaryApps",
+    statistic: "Max",
+    period: Duration.hours(24),
+    dimensions: {
+        "channel" : "stable"
+    }
+  });
+
+  const e2eAndroidMetric = new Metric({
+    metricName: "e2eCanaryTestFailure",
+    namespace: "GithubCanaryApps",
+    statistic: "Max",
+    period: Duration.hours(24),
+    dimensions: {
```

Comment:
I have a lot of TS errors in this file. e.g.

```
  Object literal may only specify known properties, but 'dimensions' does not exist in type 'MetricProps'. Did you mean to write 'dimensionsMap'?

14     dimensions: {
       ~~~~~~~~~~~~~
15         "channel" : "stable"
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
16     }
```

Also some consts are redeclared.